test/Interop/Cxx/operators/OperatorStar/Inputs/cxx-operator-star.h

```diff
@@ -0,0 +1,26 @@
+#ifndef TEST_INTEROP_CXX_STAR_OPERATORS_H
```

Comment:
I think it's better to keep all `operator*` tests in the same place. Could you please move this to the existing header `test/Interop/Cxx/operators/Inputs/member-inline.h`, and the Swift part of the test to  `test/Interop/Cxx/operators/member-inline-typechecker.swift`?

(We usually add the `-typechecker` suffix to interop test file names to make it easier to distinguish execution tests vs typechecker tests from a quick glance)

Reply:
Sure, I'll do this!